### PR TITLE
GG-19450 Added metrics for number of clusters, caches and models for "Admin panel" screen.

### PR DESCRIPTION
**Pull request by vsisko**

### CODE REVIEW

File: modules/web-console/frontend/app/components/list-of-registered-users/categories.ts [hunk 1]
```diff
@@ -21,10 +21,10 @@ export const categoriesFn = ($translate: ng.translate.ITranslateService) => [
     {name: 'Activated', visible: false, enableHiding: true},
     {name: 'Company', visible: true, enableHiding: true},
     {name: 'Country', visible: true, enableHiding: true},
-    {name: 'Last login', visible: false, enableHiding: true},
+    {name: 'Last login', visible: true, enableHiding: true},
     {name: 'Last activity', visible: true, enableHiding: true},
-    {name: 'Configurations', visible: false, enableHiding: true},
-    {name: 'Total activities', visible: true, enableHiding: true},
+    {name: 'Configurations', visible: true, enableHiding: true},
+    {name: 'Total activities', visible: false, enableHiding: true},
```
Comment (nva) [inline modules/web-console/frontend/app/components/list-of-registered-users/categories.ts]: Why you change this?

Comment (vsisko) [reply modules/web-console/frontend/app/components/list-of-registered-users/categories.ts]: Total activities are hidden because they are not implemented in the Java backend.
Configuration columns are shown because they are fixed and show useful data.

Comment (akuznetsov-gridgain) [reply modules/web-console/frontend/app/components/list-of-registered-users/categories.ts]: We need `Configurations` columns because it helps us to find user with 4000+ caches.
And it is annoying to make them visible by hands.

File: modules/web-console/e2e/testcafe/components/Table.js [hunk 2]
```diff
@@ -16,12 +16,13 @@
 
 import {Selector, t} from 'testcafe';
 
-const findCell = Selector((table, rowIndex, columnLabel) => {
-    const _findRowElement = (table, containerSelector) => {
-        const columnIndex = [].constructor.from(
+const findCell = Selector((table, rowIndex, columnId) => {
```
Comment (Klaster1) [inline modules/web-console/e2e/testcafe/components/Table.js]: Why did you change selector from label to id? I think human readable label is much easier to reason about. Was it causing issues for columns without text in header?

File: modules/web-console/e2e/testcafe/components/Table.js [hunk 3]
```diff
@@ -33,10 +34,14 @@ const findCell = Selector((table, rowIndex, columnLabel) => {
 
     table = table();
 
-    let row = _findRowElement(table, '.ui-grid-render-container.left');
+    let row = _findRowElement(
+        table,
+        '.ui-grid-render-container.left',
+        typeof columnId === 'string' ? columnId : -columnId
```
Comment (Klaster1) [inline modules/web-console/e2e/testcafe/components/Table.js]: What is `-columnId`?

File: modules/web-console/e2e/testcafe/components/Table.js [hunk 4]
```diff
@@ -16,12 +16,13 @@
 
 import {Selector, t} from 'testcafe';
 
-const findCell = Selector((table, rowIndex, columnLabel) => {
-    const _findRowElement = (table, containerSelector) => {
```
Comment (Klaster1) [inline modules/web-console/e2e/testcafe/components/Table.js]: I'd add a type annotation for `columnId` to match that of `findCell` method.

File: modules/web-console/e2e/testcafe/components/Table.js (hunk 2 above)
Comment (vsisko) [reply modules/web-console/e2e/testcafe/components/Table.js]: We have columns with icon only.

File: modules/web-console/e2e/testcafe/components/Table.js (hunk 4 above)
Comment (vsisko) [reply modules/web-console/e2e/testcafe/components/Table.js]: Added type descriptions.

File: modules/web-console/e2e/testcafe/fixtures/admin-panel.js [hunk 5]
```diff
@@ -64,3 +72,134 @@ test('Testing setting notifications', async(t) => {
...
+    await t.navigateTo(resolveUrl('/settings/admin'));
+
+    const emailFilterSelector = pageListOfUsers.usersTable.findFilter('Email');
+    const emailCellSelector = pageListOfUsers.usersTable.findCell(0, 'Email');
+
+    await t.typeText(emailFilterSelector, t.ctx.email, {paste: true})
+        .expect(pageListOfUsers.usersTable.findCell(0, 'User').textContent).contains(`${newFirstName} ${newLastName}`)
+        .expect(emailCellSelector.textContent).contains(t.ctx.email)
+        .expect(pageListOfUsers.usersTable.findCell(0, 'Company').textContent).contains(newCompaty)
+        .expect(pageListOfUsers.usersTable.findCell(0, 'Country').textContent).contains('USA');
```
Comment (MaxLosevskoy) [inline modules/web-console/e2e/testcafe/fixtures/admin-panel.js]: Why do we check with `contains`, not `equals`?

File: modules/web-console/e2e/testcafe/fixtures/admin-panel.js [hunk 6]
```diff
@@ -64,3 +72,134 @@ test('Testing setting notifications', async(t) => {
...
+    await t.typeText(pageProfile.email.control, newEmail, {replace: true, paste: true})
+        .click(pageProfile.saveChangesButton)
+        .expect(successNotification.withText('Profile saved.').exists).ok();
+
+    t.ctx.email = newEmail;
+
+    await t.navigateTo(resolveUrl('/settings/admin'));
+
+    await t.typeText(emailFilterSelector, newEmail, {paste: true})
+        .expect(emailCellSelector.textContent).contains(t.ctx.email)
```
Comment (MaxLosevskoy) [inline modules/web-console/e2e/testcafe/fixtures/admin-panel.js]: So there's no check for changing fields other than **email**?

File: modules/web-console/e2e/testcafe/fixtures/admin-panel.js (hunk 5 above)
Comment (Klaster1) [reply modules/web-console/e2e/testcafe/fixtures/admin-panel.js]: @MaxLosevskoy `equals` will most likely contain extra characters like newlines and spaces.
